### DISCUSSION TITLE
What if i found a security vulnerability on pinata.cloud's assets, im sure such big projects have a bug bounty program, isnt it ?

### DISCUSSION THREAD

**Discussion by jainiresh** [PinataCloud/docs#27] [Q&A] [answered]
Do pinata.cloud have their own bug bounty program, where i could submit serious security vulnerabilities.

**Comment by stevedylandev** [comment, marked as answer]
Hey there! We do have a bug bounty program but it is only open to members of the Hacker One community. If you have a Hacker One account please email matt@pinata.cloud. Thank you! 🙏🏻

**Reply by jainiresh** [reply]
Thanks . just emailed .
